src/py42/exceptions.py

```diff
@@ -237,6 +237,14 @@ def __init__(self, exception, user_id, list_name):
         super(Py42NotFoundError, self).__init__(exception, message)
 
 
+class Py42InvalidRule(Py42NotFoundError):
```

Comment:
class should end in `Error`

Reply:
was `Py42InvalidRuleOperationError` intended for this? It seems the same

Reply:
yea, we can merge them into one with proper messages being passed from respective module, at present, as per my understanding, the message in `py42InvalidRuleOperationError` doesn't fit this use case.

Added `Error`, renamed.